_data/apis.yml

```diff
@@ -61,12 +70,16 @@ nvstrings:
   cllink: https://github.com/rapidsai/custrings/blob/master/CHANGELOG.md
   desc: 'nvStrings (the Python bindings for [cuStrings](https://github.com/rapidsai/custrings)), provides a pandas-like API that will be familiar to data engineers & data scientists, so they can use it to easily accelerate their workflows without going into the details of CUDA programming.'
   stable:
-    - name: 'nvStrings v0.3'
-      link: https://rapidsai.github.io/projects/nvstrings/en/0.3.0/api.html
+    - name: 'nvStrings v0.7'
```

Comment:
Currently have the nvStrings docs in the `0.4.0` (matching cuStrings version) folder.

Reply:
Should be moved to 0.7.0. nvstrings is jumping from v0.3.0 to v0.7.0 in this release.